src/NewsService.Business/GetNewsByIdCommand.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LT.DigitalOffice.NewsService.Business.Interfaces;
+using LT.DigitalOffice.NewsService.Data.Interfaces;
+using LT.DigitalOffice.NewsService.Mappers.ResponsesMappers.Interface;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Business
+{
+    /// <summary>
+    /// Represents command class in command pattern. Provides method for getting user model by id.
```

Comment:
Do we need to make comments like this in the realization of interfaces? I did not see it before